test/test_turbomole.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Unit testing for Turbomole_class"""
+
+import numpy as np
+import os
+import shutil
+import unittest
+import sys
+from pathlib import Path
+import mudslide
+import yaml
+
+from mudslide.models import TMModel
+from mudslide.tracer import YAMLTrace
+
+
+
+class TestTMModel(unittest.TestCase):
+    """Test Suite for TMModel class"""
+    
+
+    def setUp(self):
+        self.turbomole_files = ["control"]
+        for fl in self.turbomole_files:
+            shutil.copy("turbomole_files/"+fl,".")
```

Comment:
This `turbomole_files` directory does not appear anywhere on the repo.

Reply:
added now

Reply:
Right now this only works if you run `pytest` in the `test/` directory. But you should be able to run `pytest` from the base directory as well (or potentially anywhere). You should be able to accomplish that by having the code look for the location of the present test being run (which is different from the current location) and use that as a base directory.